chapter0/index.md

```diff
@@ -4,13 +4,13 @@ leadImage: illustration_00.jpg
 
 # Welcome - Easy EdgeDB
 
+Welcome to Easy EdgeDB! It's an interactive, illustrated textbook designed as a one-stop shop for learning EdgeDB.
```

Comment:
It's not super interactive yet. Maybe just "illustrated"?

Reply:
Haha. This isn't interactive yet. I got confused when I couldn't execute queries as well. Illustrated sounds nice.

Reply:
```suggestion
Welcome to Easy EdgeDB! It's an illustrated textbook designed as a one-stop shop for learning EdgeDB.
```